backend/src/Good/good.service.ts

```diff
@@ -31,6 +31,57 @@ class Service {
         }
     }
 
+    /**
+     * Get total expenses
+     */
+    public async getExpense() {
```

Comment:
Hmm interesting approach but I'm not sure it will work though. Ex: Let's say we have 3 bike at 1000$ each, our expense will be 3000$. What if we sell a bike? The quantity will be 2 which will give us an expense of 2000$ which is not true, I might be wrong but let me know what you think.

Reply:
Yea, I understand. Once we sell a bike, we lose its expense. Should we replace it with the manufacturing orders?

Reply:
Yes, the expenses should be on the completed manufacturing orders and the income should be on the completed client orders

Reply:
but, when we order raw goods, for example, those are also expenses, shouldn't we add them ?

Reply:
Manufacturing orders contain raw goods. So they are included.